interpreter/syntax/ast.ml

```diff
@@ -264,3 +267,6 @@ let string_of_name n =
   in
   List.iter escape n;
   Buffer.contents b
+
+let contains_null_ref l = List.exists (fun elem -> elem.it = RefNull) l
```

Comment:
For hygiene, I'd prefer not to dump random helper function in the AST module. Given how trivial the function is, it seems fine to duplicate it in the two modules that use it.

Reply:
Done.